docs/mautic_community_portal/mautic_community_portal.rst

```diff
@@ -0,0 +1,16 @@
+Mautic community portal
+#######################
+
+The community portal exists at :xref:`community-portal`.
+
+It's the central place where everything relating to the community happens. Everything from proposing and organizing meetup groups, teams, and working groups to voting on the next members of the Mautic council happens on the portal.
+
+.. vale off
+
+It's integrated with a single sign-on service, so you can use the same username and password as you use on the forums.
+
+Members of Mautic have extra privileges such as being able to propose in elections and vote. :xref:`open-collective`, and your email will be synchronized with the system so that you can claim your membership via your user profile.
```

Comment:
```suggestion
Members of Mautic have extra privileges such as being able to stand in elections and vote. :xref:`open-collective`, and you're added to the membership list so that you can claim your membership via your user profile. Note that this is a manual process, so it might take a little time.
```
Active voice, also please remove references to open collective here, because all membership signups should be via https://mau.tc/membership.